android/src/main/java/com/swmansion/reanimated/nodes/TransformNode.java

```diff
@@ -23,7 +23,7 @@
 
     @Override
     public double getValue(NodesManager nodesManager) {
-      return nodesManager.getNodeValue(nodeID);
+      return (double) nodesManager.getNodeValue(nodeID);
```

Comment:
transforms can have strings too (also add to one of example screens to see if `rotate` works ok)